### DISCUSSION TITLE
I deleted the project, but the link still works

### DISCUSSION THREAD

**Discussion by MaSStiK** [vercel/community#6465] [Help] [answered]
### Summary

About half a year ago I created a repository on Github and posted a page on Vercel, but a little later I deleted it, and now I noticed that the page on the link is still working. Is there any way to completely remove it?

Link to website https://punkt-b.vercel.app

### Steps to Reproduce

At the moment this project is not displayed in my account
![image](https://github.com/vercel/community/assets/72652605/afc157f2-a04b-442c-8b08-dece4c78a4f9)

**Comment by amyegan** [comment, marked as answer]
Hey @MaSStiK. That subdomain is in use on a project that belongs to someone else (or an alternate account of yours)
